src/pvq_encoder.c

```diff
@@ -418,15 +433,20 @@ static int pvq_theta(od_coeff *out, const od_coeff *x0, const od_coeff *r0,
     od_val16 xr[MAXN];
     int gain_bound;
     int prev_k;
+    pvq_search_item items[20];
+    int idx;
+    int nitems;
+    idx = 0;
     gain_bound = OD_SHR(cg - gain_offset, OD_CGAIN_SHIFT);
     /* Perform theta search only if prediction is useful. */
     theta = OD_ROUND32(OD_THETA_SCALE*acos(corr));
     m = od_compute_householder(r16, n, gr, &s, rshift);
     od_apply_householder(xr, x16, r16, n);
     prev_k = 0;
     for (i = m; i < n - 1; i++) xr[i] = xr[i + 1];
-    /* Search for the best gain within a reasonable range. */
-    for (i = OD_MAXI(1, gain_bound - 1 + speed); i <= gain_bound + 1;
```

Comment:
Did you intend to revert https://github.com/xiph/daala/pull/196/commits/4b4d326c59d4344896934e149c7242ffc80d6405 in this commit ? Don't we want to still test fewer gain/thetas at speed=1? Otherwise we should just drop https://github.com/xiph/daala/pull/196/commits/4b4d326c59d4344896934e149c7242ffc80d6405 from this PR